test/tests/interfaces/pbs_preempt_params.py

```diff
@@ -48,13 +48,8 @@ class TestPreemptParamsQmgr(TestInterfaces):
     error_unauth = "Unauthorized Request"
     error_code_unauth = "15007"
 
-    def test_set_unset_preempt_queue_prio(self):
-        """
-        This test case sets preempt_queue_prio parameter to valid/invalid
-        values and checks if the server allows/disallows the operation.
-        """
-        msg = "preempt_queue_prio set to invalid value"
-        a = {'preempt_queue_prio': 'abc'}
+    def common_tests(self, param, msg):
```

Comment:
It isn't necessary, but consider adding a doc string to help the reader understand what this function is doing.